src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -87,26 +96,46 @@ export default {
 			default: 0
 		},
 		/**
-		 * if true, it displays the message author on top of the message.
+		 * The message id.
+		 */
+		id: {
+			type: Number,
+			required: true
+		},
+		/**
+		 * If true, it displays the message author on top of the message.
 		 */
 		showAuthor: {
 			type: Boolean,
 			default: true
 		},
 		/**
-		 * Style the message as a quote.
+		 * Specifies if the message is temporary in order to display the spinner instead
+		 * of the message time.
 		 */
-		isQuote: {
-			type: Boolean,
-			default: false
-		},
 		isTemporary: {
 			type: Boolean,
 			required: true
 		},
+		/**
+		 * Specifies if the message is the first of a group of same-author messages.
+		 */
 		isFirstMessage: {
 			type: Boolean,
 			required: true
+		},
+		/**
+		 * The conversation token.
+		 */
+		token: {
+			type: String,
+			required: true
+		},
+		/**
+		 * The parent message's id.
+		 */
+		parent: {
+			type: Number
```

Comment:
required/default

Reply:
It's non of the 2:

* It's not required because not every message has a parent;
* Doesn't make sense to have a default parent :)

Reply:
Doesn't eslint screams? :stuck_out_tongue_closed_eyes:

Reply:
Surprisingly it doesn't :D